src/Scraper/Application/XpathFinder.php

```diff
@@ -43,11 +45,13 @@ public function extract(string $url, $configs): array
                 );
             }
 
-            $result[$config['name']] = $subcrawler->each(function ($node) {
+            $result['data'][$config['name']] = $subcrawler->each(function ($node) {
```

Comment:
I think `result` is becoming a complex data structure (maybe it has always been so). Should we think about making it a value object so it's easier to reason with?

Reply:
Yes, I was thinking about create a value object for this, but I won't be backward compatible, so I prefer for now to let it like this and do it backwards compatible. For the definitive version we could do it. I'm going to create an issue for this.